docs(mongodb): document Windows limitation for Percona mongodb_exporter (#25041)
newrelic/docs-website: src/install/mongodb/windows/install-msi.mdx

headingText: Download using MSI
---

<Callout variant="important">

>   Windows builds of this integration are frozen at their current bundled version of the Percona `mongodb_exporter`. Upstream Percona stopped producing Windows binaries from `v0.47.1` onward, so updates to the exporter, including security fixes, aren't available for Windows until Windows builds resume upstream.

Reviewer: ```suggestion
  Windows builds of this integration are frozen at their current bundled version of the Percona `mongodb_exporter`. Upstream Percona stopped producing Windows binaries starting with `v0.47.1`. Consequently, updates to the exporter, including security fixes, are unavailable until upstream builds resume.
```
